docs/readme.windows.txt

```diff
@@ -24,6 +24,9 @@ Running  RavenDB            RavenDB
 To manage service you can use Stop-Service and Start-Service cmdlets (requires administrator privileges).
 
 * Setup
-    Open browser, if not opened automatically, at url printed in "Server available on: <url>"
-    Follow the web setup instructions at: https://ravendb.net/docs/article-page/4.0/csharp/start/installation/setup-wizard
+Open browser, if not opened automatically, at url printed in "Server available on: <url>"
+Follow the web setup instructions at: https://ravendb.net/docs/article-page/4.0/csharp/start/installation/setup-wizard
```

Comment:
`/4.0/` ? Maybe better to use `/latest/` in all places?

Reply:
https://github.com/ravendb/ravendb/commit/1f49d02bae103139338664bcfb91a4e65f5b4220